src/tcgwars/logic/impl/gen4/SupremeVictors.groovy

```diff
@@ -629,132 +795,159 @@ public enum SupremeVictors implements LogicCardInfo {
+        return basic (this, hp:HP100, type:R, retreatCost:3) {
           weakness W
           resistance F, MINUS20
           move "Flame Jet", {
-            text "Flip a coin. If heads, this attack does 40 damage to 1 of your opponent’s Pokémon."
+            text "Flip a coin. If heads, this attack does 40 damage to 1 of your opponent's Pokémon. (Don't apply Weakness and Resistance for Benched Pokémon.)"
             energyCost R, C
             attackRequirement {}
             onAttack {
-              damage 0
+              flip {
+                damage 40, opp.all.select("Choose a Pokémon to deal 40 damage to")
+              }
             }
           }
           move "Heat Blast", {
             text "50 damage. "
             energyCost R, C, C
             attackRequirement {}
             onAttack {
-              damage 0
+              damage 50
             }
           }
-
         };
       case CHIMECHO_21:
-        return basic (this, hp:HP070, type:PSYCHIC, retreatCost:1) {
+        return basic (this, hp:HP070, type:P, retreatCost:1) {
           weakness P, PLUS20
           move "Heal Bell", {
             text "Remove 3 damage counters from each of your Pokémon."
             energyCost C
-            attackRequirement {}
+            attackRequirement {
+              assert my.all.find{it.numberOfDamageCounters()} : "Your Pokémon are healthy"
+            }
             onAttack {
-              damage 0
+              my.all.each {
+                heal 30, it
+              }
             }
           }
           move "Super Psywave", {
-            text "Choose 1 of your opponent’s Pokémon. Count the amount of Energy attached to that Pokémon. Put"
+            text "Choose 1 of your opponent's Pokémon. Count the amount of Energy attached to that Pokémon. Put that many damage counters on the Pokémon."
             energyCost P
-            attackRequirement {}
+            attackRequirement {
+              assert opp.all.find{it.cards.energyCount(C)} : "None of your opponent's Pokémon have any Energy attached"
+            }
             onAttack {
-              damage 0
+              def tar = opp.all.findAll{it.numberOfDamageCounters()}.select("Choose 1 of your oppopnent's Pokémon")
+              directDamage 10*tar.numberOfDamageCounters(), tar
             }
           }
-
         };
       case CLAYDOL_22:
-        return evolution (this, from:"Baltoy", hp:HP090, type:PSYCHIC, retreatCost:1) {
+        return evolution (this, from:"Baltoy", hp:HP090, type:P, retreatCost:1) {
           weakness P, PLUS20
           move "Antique Magic", {
-            text "in any way you like."
+            text "Put 3 damage counters on any Pokémon (both yours and your opponent's) in any way you like."
             energyCost C, C
             attackRequirement {}
             onAttack {
-              damage 0
+              putDamageCountersOnOpponentsPokemon(3,all)
```

Comment:
> any Pokémon (both yours and your opponent's)

Reply:
The method is poorly named imo. It accepts any PCSList as its target space, and in this case all allows you to choose both your Pokémon and your opponents Pokémon.

Reply:
Neat. wasn't aware there was an all() static.